LICENSE.md

```diff
@@ -424,6 +424,31 @@ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+
+### ESLint
```

Comment:
Is this required here?  Do we actually ship ESLint as part of the Cesium JavaScript library?